README.md

```diff
@@ -139,63 +139,66 @@ mention `std::` or C++. For example, "`<type_traits>`: `is_cute` should be true
 It's okay if you report an apparent STL bug that turns out to be a compiler bug or surprising-yet-Standard behavior.
 Just try to follow these rules, so we can spend more time fixing bugs and implementing features.
 
+# Visual Studio Installer Prerequisites
+
+1. Install [VS 2026 Insiders][] and keep it up to date.
+    + **You must install Insiders for STL development.** The STL and the compiler ship together, and we frequently
+    need the latest compiler fixes and features, so the last production release of the compiler is too old.
+2. Select the "Desktop development with C++" workload.
+3. Select the following components at a minimum:
+    + "MSVC v145 - C++ x64/x86 build tools (Latest)"
+    + "C++ CMake tools for Windows"
+    + "C++ AddressSanitizer"
+    + "Windows 11 SDK (10.0.26100)" or later
+    + "C++ Clang tools for Windows (20.1.8 - x64/x86)"
+    + *Optional:* "MSVC v145 - C++ ARM64/ARM64EC build tools (Latest)"
+4. Install [Python][] 3.13 or later.
+    + Select "Add python.exe to PATH" if you want to follow the instructions below that invoke `python`.
+    Otherwise, you should be familiar with alternative methods.
+
 # How To Build With The Visual Studio IDE
 
-1. Install Visual Studio 2022 17.14.12 Preview 1 or later.
-    * Select "Windows 11 SDK (10.0.26100.4188)" in the VS Installer.
-    * Select "MSVC v143 - VS 2022 C++ ARM64/ARM64EC build tools (Latest)" in the VS Installer
-    if you would like to build the ARM64/ARM64EC target.
-    * We recommend selecting "C++ CMake tools for Windows" in the VS Installer.
-    This will ensure that you're using supported versions of CMake and Ninja.
-    * Otherwise, install [CMake][] 3.31.0 or later, and [Ninja][] 1.12.1 or later.
-    * Make sure [Python][] 3.13 or later is available to CMake.
-2. Open Visual Studio, and choose the "Clone or check out code" option. Enter the URL of this repository,
+1. Open Visual Studio, and choose the "Clone or check out code" option. Enter the URL of this repository,
    `https://github.com/microsoft/STL`.
-3. Open a terminal in the IDE with `` Ctrl + ` `` (by default) or press on "View" in the top bar, and then "Terminal".
-4. In the terminal, invoke `git submodule update --init --progress`
-5. Choose the architecture you wish to build in the IDE, and build as you would any other project. All necessary CMake
+2. Open a terminal in the IDE with `` Ctrl + ` `` (by default) or press on "View" in the top bar, and then "Terminal".
+3. In the terminal, invoke `git submodule update --init --progress`

```

Comment:
This was a no-op for me.

Reply:
Yep, it's clearly cloning the submodules by default, which is good news for us. Fixing.